ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/ParsePayloadTest.java

```diff
@@ -36,7 +36,7 @@ public void testOutput() {
         .apply("AddAttributes", MapElements.into(TypeDescriptor.of(PubsubMessage.class))
             .via(element -> new PubsubMessage(element.getPayload(), ImmutableMap.of(
                 "document_namespace", "test", "document_type", "test", "document_version", "1"))))
-        .apply(ParsePayload.of(schemasLocation));
+        .apply(ParsePayload.of(schemasLocation, null));
```

Comment:
instead of nulls, I'd prefer we pass in a value provider via `pipeline.newProvider` with a null value, or if that doesn't work a `StaticValueProvider.of(null)`.